PED/SetPedModelIsSuppressed.md

```diff
@@ -5,11 +5,11 @@ ns: PED
 
 ```c
 // 0xE163A4BCE4DE6F11 0x7820CA43
-void SET_PED_MODEL_IS_SUPPRESSED(Ped ped, BOOL toggle);
+void SET_PED_MODEL_IS_SUPPRESSED(Hash model, BOOL toggle);
```

Comment:
Please change the `Hash` to an `int` to keep backwards compatibility for C# resources.

Reply:
```suggestion
void SET_PED_MODEL_IS_SUPPRESSED(int model, BOOL toggle);
```